docs/proposals/signal-api.md

```diff
@@ -0,0 +1,174 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Signal API
+
+This document describes signal API effort in Kubernetes.
+
+## Abstract
+
+The goal is to provide a possibility to send signals to container PID 1 processes. At the beginning, we
```

Comment:
OK, good point.

Regarding rkt - from what I know, machinectl also has a kill feature, to which you can provide a PID. But in case of VM flavor, we'll need to get a PID of the "entrypoint applications". I'll try to think about this problem, but suggestions from rktnetes devs are more than welcome.